### PR TITLE
ci: use lotus release/v1.20.0 and test uniswap

### PR DESCRIPTION
**Pull request by ychiaoli18**

### CODE REVIEW

File: node/Makefile [hunk 1]
```diff
@@ -5,15 +5,21 @@ LOTUS_PATH:=extern/lotus
...
-	git submodule set-branch -b feat/nv18-fevm node/extern/lotus
+	git submodule set-branch -b release/v1.20.0 node/extern/lotus
 	git submodule sync
 	git submodule update --init --recursive --remote
+
+gomod:
+	@echo "module github.com/filecoin-project/fevm-contract-tests/node" > go.mod
+	@echo "go 1.18" >> go.mod
+	@echo "replace github.com/filecoin-project/lotus => ./extern/lotus" >> go.mod
+	@echo "replace github.com/filecoin-project/filecoin-ffi => ./extern/lotus/extern/filecoin-ffi" >> go.mod
```
Comment (raulk) [inline node/Makefile:16]: Can use `go mod edit` as it wil perform validations for you, but this is fine too.

File: .circleci/config.yml [hunk 2]
```diff
@@ -140,12 +140,43 @@ jobs:
...
+          no_output_timeout: 30m
+      - run:
+          command: ./node/bin/node
+          background: true
+      - run:
+          command: cp .env ./extern/fevm-uniswap-v3-core/
+      - run:
+          command: cd ./extern/fevm-uniswap-v3-core/ && yarn
+      - run:
+          command: cd ./extern/fevm-uniswap-v3-core/ && npx hardhat --network itest test || true
```
Comment (raulk) [inline .circleci/config.yml:172]: This PR also adds a Uniswap target to CI -- but that doesn't seem to be captured in the PR description. Is this intended?

Comment (ychiaoli18) [reply .circleci/config.yml]: I changed the PR description and updated the uniswap testing repo to make everything work now.

File: node/Makefile (hunk 1 above)
Comment (ychiaoli18) [reply node/Makefile:16]: ok, I'll leave it as is and change it later if necessary.

File: kit/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const request = require("sync-request");
+const { ethers } = require("ethers");
+
+// Setup testing environment
+const nodeManagerUrl = "http://localhost:8090";
+
+function initNode(filAmount) {
+  if (process.argv.includes("itest")) {
```
Comment (raulk) [inline kit/index.js:8]: Returning early on the negative is more idiomatic.

File: node/main.go [hunk 4]
```diff
@@ -35,6 +35,7 @@ func initKit() {
 		kit.ThroughRPC(),
 		kit.WithCfgOpt(func(cfg *config.FullNode) error {
 			cfg.ActorEvent.EnableRealTimeFilterAPI = true
```
Comment (raulk) [inline node/main.go:37]: This option is now gone. It is now a negative (Disable...) and lives under the Fevm group. You can remove this line.

**Review by raulk — APPROVED** [review]

**Pull request merged**